### PR TITLE
Change favicon

### PR DESCRIPTION
**Pull request by DEJENERUTA**

### CODE REVIEW

**Comment by bence-toth** [comment]
I believe the following should be updated as well:
- `logo192.png`
- `logo512.png`

I'm also wondering if we should ditch the `favicon.ico` and just use the `logo192.png` version instead.

Finally, why is the favicon black and white? Shouldn't we use transparent + Persian green (the logo color)?

**Comment by DEJENERUTA** [comment]
This commit updates the following files as requested:
- Updated logo192.png and logo512.png to the latest version.
- Replaced favicon.ico with logo192.png for consistency.

File: public/manifest.json [hunk 1]
```diff
@@ -2,11 +2,6 @@
...
   "name": "Create React App Sample",
   "icons": [
-    {
-      "src": "favicon.ico",
-      "sizes": "64x64 32x32 24x24 16x16",
-      "type": "image/x-icon"
-    },
     {
       "src": "logo192.png",
       "type": "image/png",
```
Comment (bence-toth) [inline public/manifest.json:7]: `"sizes"` here should be `192x192 64x64 32x32 24x24 16x16`

File: public/index.html [hunk 2]
```diff
@@ -2,18 +2,14 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/logo192.png" />
```
Comment (bence-toth) [inline public/index.html]:
```suggestion
    <link rel="icon" type="image/png" href="%PUBLIC_URL%/logo192.png" />
```

**Review by bence-toth — CHANGES_REQUESTED** [review]

**Review by bence-toth — APPROVED** [review]

**Pull request merged**